onnxruntime/python/tools/transformers/fusion_skiplayernorm.py

```diff
@@ -21,6 +21,13 @@ def __init__(self, model: OnnxModel):
 
     def fuse(self, node, input_name_to_nodes, output_name_to_node):
         add = self.model.get_parent(node, 0, output_name_to_node)
+
+        # In some models there is input_ids->gather->add->LayerNorm and one of input of the
+        # add node is initializer with fixed shape which should not be fused into SkipLayerNorm
+        for add_input in add.input:
```

Comment:
How about returning when number of inputs of Add node is not 2?

Reply:
That makes sense